Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
@UniqMartin It's not "on a whim", it's because multiple users have been confused by this. As you've said: GitHub provides redirects.

@DomT4 `formulae-patches` is perhaps my preference given these are never patches for core itself.